src/interface/src/app/plan/area-scrolling-notes/area-scrolling-notes.component.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { AuthService, PlanNotesService } from '@services';
-import { MatLegacyDialogModule as MatDialogModule } from '@angular/material/legacy-dialog';
-import { MatLegacySnackBarModule as MatSnackBarModule } from '@angular/material/legacy-snack-bar';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
```

Comment:
did you test we are not getting any of those annoying errors about conflicting versions with old vs new dependencies?